browser/ui/views/tabs/brave_tab.h

```diff
@@ -59,6 +62,8 @@ class BraveTab : public Tab {
   std::unique_ptr<ui::Layer> shadow_layer_;
   gfx::FontList normal_font_;
   gfx::FontList active_tab_font_;
+
+  static constexpr int kExtraLeftPadding = 4;
```

Comment:
I was hoping we can verify this value is the same value in the tab.cc file, but I can't think of it without touching chromium_src/ overriding.